media-gfx/exiv2/exiv2-0.27.7.recipe

```diff
@@ -82,6 +81,7 @@ BUILD()
 
 	cmake -Bbuild -S. -DCMAKE_BUILD_TYPE=Release \
 		$cmakeDirArgs \
+		-DCMAKE_INSTALL_DOCDIR=$documentationDir/packages/exiv2 \
```

Comment:
@jmairboeck since we now create 2 exiv2_doc versions, maybe disable this one in favor or the newer?

Reply:
Yes, I guess this could be removed, if this is only retained for backwards compatibility.

Can this recipe maybe be renamed to exiv2_0.27 and the new one without a suffix, if it is going to be the default version?

Reply:
Renaming this shouldn't be a major problem, maybe also remove the devel package for this one too so we wouldn't have to set a version string for packages depening on this?